homework-submission/todo-app/src/component/week3Components/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
+import Name from "./name";
+import Img from "./img";
+import Region from "./region";
+import Gender from "./gender";
+import "core-js";
+
+function Week3Components() {
+  const [data, setData] = useState("");
```

Comment:
`data` is a quite generic name, try to give this a more descriptive name, of what the variable(s) actually contain.

Reply:
Changed to user.